enterprise_access/apps/api/tasks.py

```diff
@@ -127,6 +130,50 @@ def send_notification_email_for_request(
     )
 
 
+@shared_task(base=LoggedTaskWithRetry)
+def send_cancel_email_for_pending_assignment(cancelled_assignment):
+    """
+    Send email via braze for cancelling pending assignment
+
+    Args:
+        cancelled_assignment: (string) the subsidy request uuid
+    """
+    learner_content_assignment_action = LearnerContentAssignmentAction(
+        uuid=cancelled_assignment.uuid, assignment=cancelled_assignment, action_type=AssignmentActions.CANCELLED
+    )
```

Comment:
You don't want to set the `uuid` here (and you wouldn't want it to share the uuid of the related Assignment, anyway) - the model will create a new one by default for you.

Reply:
That makes sense. Removed the uuid here.